src/test/components/CallTreeSidebar.test.js

```diff
@@ -181,12 +181,12 @@ describe('CallTreeSidebar', function () {
       funcNamesDict: { A, B, C },
     } = setup(getProfileWithSubCategories());
     selectNode([A, B, C]);
-    expect(queryByText('FakeSubCategoryC')).not.toBeInTheDocument();
```

Comment:
This test's title was `can expand subcategories` which is not true anymore since they are exanded by default. Could you change it to something like `can toggle subcategories`?

Reply:
The categories are not expanded by default anymore with my newest change.